common/src/main/java/com/wynntils/features/user/GammabrightFeature.java

```diff
@@ -27,26 +27,24 @@ public class GammabrightFeature extends UserFeature {
 
     @RegisterKeyBind
     private final KeyBind gammabrightKeyBind =
-            new KeyBind("Gammabright", GLFW.GLFW_KEY_G, true, this::onGammabrightKeyPress);
+            new KeyBind("Gammabright", GLFW.GLFW_KEY_G, true, this::applyGammabright);
 
     @SubscribeEvent
     public void onWorldStateChange(WorldStateEvent event) {
         if (event.getNewState() != WorldStateManager.State.WORLD) return;
-        if (!gammabrightEnabled) return;
 
-        lastGamma = McUtils.options().gamma;
-        McUtils.options().gamma = 1000d;
+        applyGammabright();
     }
 
-    private void onGammabrightKeyPress() {
+    private void applyGammabright() {
```

Comment:
`applyGammabright` seems to be a toggle of `gammabrightEnabled`. As such, why are we calling it on world state change, which toggles the gamma as the world changes, and when the config is changed, which negates the change?

Perhaps make a method 
```java
private void updateGamma(boolean bright) {
            if (bright) {
                McUtils.options().gamma = lastGamma;
            } else {
                lastGamma = currentGamma;
                McUtils.options().gamma = 1000d;
            }
}
```

which is called individually?

Reply:
Right, thanks